packages/integrations/README.md

```diff
@@ -2,7 +2,65 @@
 
 Configu SDK for Node.js published to [npm](https://www.npmjs.com/package/@configu/node).
 
-## Install
+
+# Integrations Package
+
+## What is an Integration?
+
+An integration in Configu is a module that allows Configu to interact with external systems, services, or tools. Integrations enable seamless management and collaboration on software configuration data across various environments and systems. They provide a standardized way to connect Configu with other platforms, ensuring consistent and efficient configuration operations.
+
+## How to Use an Integration in the New CLI Syntax
+
+Using an integration in the new CLI syntax is straightforward. Here is an example of how to use an integration:
+
+```sh
+configu integration <integration-name> <command> [options]
```

Comment:
this is not how the integrations work.
there's not a specific command to use the integrations, because actually everything in configu is a sort of integration.
for example:

when you can a command such as:
```sh
configu upsert \
  --store 'configu' --set 'development' --schema './start.cfgu.json' \
  --config "GREETING=hey" --config "SUBJECT=<YOUR_NAME>"
```

the "configu" store runs the class of `ConfiguConfigStore` to set the config specified in this command inside configu.
if you switch to "aws-parameter-store", it will use the `AwsParameterStoreConfigStore` class, which also has get and set functions to store configs on was.

on the same command, you have a path to a schema file.
inside that schema - you can write test expressions to using the syntax of Microsoft's `adaptive-expressions` library.
if you want to write your own custom validation functions and use them inside that test, you can provide js files that will include your own functions, and then use them in your validation tests.
that's also a kind of integration, that allow you to enrich the test expressions with your own functions.

at the end, assuming this is your next command:
```sh
configu eval \
  --store 'configu' --set 'development/qa' --schema './start.cfgu.json' \
| configu export \
  --format "Dotenv" \
> ".env"
```

this command will get the configs from the "configu" store (using the custom store class written in the integrations), but it can also use any of your custom classes to retrieve configs from different places, using your custom implementation.
this command is also exporting the configs in a Dotenv format.
this format is also an integration! we have a function that can retrieve an object, and return a string that answers the rules of a Dot Env file.
you can also implement a custom integration that will retrieve the config object, and return a custom result, e.g. - a PHP file with hard-coded variables that should be injected into your machine. if you need such an integration for yourself - you can write it. configu will use it.